### PR TITLE
Appsettings pipeline update

### PR DESCRIPTION
**Pull request by cbardini**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ This client uses the OAuth2/OIDC Authorization Code Flow + PKCE to obtain an acc
 
 - Register an Authorization Code client in OCS and ensure that the registered client in OCS contains `https://127.0.0.1:54567/signin-oidc` in the list of RedirectUris.
 - Configure the sample using the file [appsettings.placeholder.json](AuthorizationCodeFlow/appsettings.placeholder.json). Before editing, rename this file to `appsettings.json`. This repository's `.gitignore` rules should prevent the file from ever being checked in to any fork or branch, to ensure credentials are not compromised.
-- Replace the placeholders in the `appsettings.json` file with your Tenant Id, Client Id, and Client Secret obtained from registration.
+- Replace the placeholders in the `appsettings.json` file with your Tenant Id, Client Id, username, and password obtained from registration.
```
Comment (derekendres) [inline README.md:27]: maybe clearer here that username/password is only for automated test

**Review by derekendres — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (cbardini) [reply README.md]: Updated the instructions to mention that the username/password are only used for testing

File: AuthorizationCodeFlow/appsettings.placeholder.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+  "TenantId": "PLACEHOLDER_REPLACE_WITH_TENANT_ID",
+  "ClientId": "PLACEHOLDER_REPLACE_WITH_CLIENT_ID",
   "ApiVersion": "v1",
   "RedirectHost": "https://127.0.0.1",
   "RedirectPort": "54567",
   "RedirectPath": "signin-oidc",
-  "Username": "TESTONLY_REPLACE_WITH_USERNAME",
-  "Password": "TESTONLY_REPLACE_WITH_PASSWORD"
+  "Username": "PLACEHOLDER_REPLACE_WITH_USERNAME",
+  "Password": "PLACEHOLDER_REPLACE_WITH_PASSWORD"
```
Comment (kelseyduffy) [inline AuthorizationCodeFlow/appsettings.placeholder.json:10]: along the same line as Derek's comment, changing the strings here to say testonly would be useful too

File: azure-pipelines.yml [hunk 3]
```diff
@@ -38,12 +34,17 @@ jobs:
...
+        value: $(AuthorizationUsername)
+      - name: Password
+        value: $(AuthorizationPassword)
+    steps:        
+      - template: '/miscellaneous/build_templates/appsettings.yml@templates'
         parameters:
-          fileOriginal: $(CredFile)
-          fileLibrary: $(CredFileLibrary)
-          filePath: /AuthorizationCodeFlow
+          secrets: 'TenantId, NamespaceId, AuthorizationUsername, AuthorizationPassword, AuthorizationCodeClientId'
```
Comment (kelseyduffy) [inline azure-pipelines.yml:47]: where is it pulling the tenantid and namespaceid from?

File: README.md (hunk 1 above)
Comment (cbardini) [reply README.md]: I reworded this section

File: azure-pipelines.yml (hunk 3 above)
Comment (cbardini) [reply azure-pipelines.yml:47]: This is pulled from the Azure Key Vault

File: AuthorizationCodeFlow/appsettings.placeholder.json (hunk 2 above)
Comment (cbardini) [reply AuthorizationCodeFlow/appsettings.placeholder.json]: I changed the placeholder to include ```_TEST_ONLY```

**Review by derekendres — APPROVED** [review]

**Pull request merged**